types/block.go

```diff
@@ -660,7 +659,6 @@ func (data *EvidenceData) StringIndented(indent string) string {
 %s}#%v`,
 		indent, strings.Join(evStrings, "\n"+indent+"  "),
 		indent, data.hash)
-	return ""
```

Comment:
My local lint was failing with this being here. Figured it wasn't worth making a separate PR to fix this.